node-fs-readfile/cat.js

```diff
@@ -0,0 +1,22 @@
+import { readFile } from 'node:fs/promises';
+
+async function readCatAnyFiles() {
+  const fileArray = process.argv.slice(2);
```

Comment:
It's best not to put the data type in the variable name. The reader can usually easily figure out what data type the variable holds, so putting it in the name makes the code more verbose and actually harder to read. Just `files` would be a great name.

Reply:
Updated